File: solutions/Team.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d",  &n);
    int ans = 0;
    while (n--) {
        int x; 
        int cnt1 = 0;
        
        for (int i = 0; i < 3; i++) {
            scanf("%d", &x);
            if (x == 1) cnt1 += 1;
        }
        if (cnt1 >= 2) ans += 1;
    }
    printf("%d \n", ans);
    
    return(0);
}

File: solutions/Team.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();
        int ans = 0;
        while (n-- > 0) {
            int cnt_1 = 0;
            for (int i = 0; i < 3; i++) {
                int x = sc.nextInt();
                if (x == 1) cnt_1 += 1;
            }
            if (cnt_1 > 1) ans += 1;
        }
        System.out.println(ans);
    }
}